source/base/DateTime.ooc

```diff
@@ -41,6 +41,7 @@ DateTime: cover {
 	_ticks: ULong = 0
 	ticks ::= this _ticks
 	init: func@ (=_ticks)
+	init: func@ ~fromNanoseconds (nanoseconds: ULong) { this _ticks = nanoseconds / This nanosecondsPerTick }
```

Comment:
@emilwestergren yeah, I thought about that... it might be better after all to have a factory method instead

@marcusnaslund not right now